scalableminds/webknossos
# Reactify Project Creation

### Mailable description of changes:
- [Admin.Project] Update project creation form

### Steps to test:
- Create new project. Check request for correct values
- Edit an existing project. Check request for correct values
- Pause / Resume a project. 

### Issues:
- part of #2009

@daniel-wer @fm3 Attention, when merging this into brave new world skeletontracing. In this PR all project related API calls use he project name as an identifier. I believe you changed this to project `id`!?

------
